includes/utils.php

```diff
@@ -334,20 +336,9 @@ public static function get_timezone_string() {
 	 * @return string A URL for creating new post using Elementor.
 	 */
 	public static function get_create_new_post_url( $post_type = 'page', $template_type = null ) {
-		$query_args = [
-			'action' => 'elementor_new_post',
-			'post_type' => $post_type,
-		];
-
-		if ( $template_type ) {
-			$query_args['template_type'] = $template_type;
-		}
-
-		$new_post_url = add_query_arg( $query_args, admin_url( 'edit.php' ) );
-
-		$new_post_url = add_query_arg( '_wpnonce', wp_create_nonce( 'elementor_action_new_post' ), $new_post_url );
+		//_deprecated_function( __FUNCTION__, '3.0.8', 'Documents_Manager::get_create_new_post_url()' );
 
-		return $new_post_url;
+		return Documents_Manager::get_create_new_post_url( $post_type, $template_type );
```

Comment:
Use it from Plugin.

Reply:
Handled https://github.com/elementor/elementor/commit/23259ba9e457c9747a2ebf03ec3d5125c40ba9c7